test/tap/tests/mysql-protocol_compression_level-t.cpp

```diff
@@ -0,0 +1,184 @@
+#include <cstdlib>
+#include <cstdio>
+#include <cstring>
+#include <unistd.h>
+
+#include <string>
+#include <sstream>
+#include <fstream>
+#include "mysql.h"
+
+#include "tap.h"
+#include "command_line.h"
+#include "utils.h"
+
+inline unsigned long long monotonic_time() {
```

Comment:
I think we have used this function already in several tests files, would be nice to place it in `utils.h`/`utils.cpp`. So we avoid keep copying it around.

Reply:
Thanks Fixed